pkg/virt-config/config_test.go

```diff
@@ -182,15 +178,20 @@ var _ = Describe("ConfigMap", func() {
 		table.Entry("when unset, GetMemoryOvercommit should return the default", "", virtconfig.DefaultMemoryOvercommit),
 	)
 
-	table.DescribeTable(" when emulatedMachines", func(value string, result []string) {
+	table.DescribeTable(" when emulatedMachines", func(value1 string, value2 string, result []string) {
```

Comment:
probably best to name them a little bit better: 

value1 -> cpuArch
value2 -> machineKeys